charts/cluster-scanner/values.yaml

```diff
@@ -0,0 +1,136 @@
+# basic Default values for cluster-scanner.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  clusterConfig:
+    name: ""
+  sysdig:
+    accessKey: ""
+    region: "us1"
+    # apiHost is used to manually set the Sysdig backend host
+    # when no region is suitable (e.g. on-premise installations)
+    # Must have the protocol (http:// or https://)
+    # apiHost: ""
+    # skipVerifyCertificate might be used in on-premise installations.
+    skipVerifyCertificate: false
+
+  scannerMode: "local"  # or "multi"
+  loggingLevel: "INFO"
+
+eveEnabled: false
+eveIntegrationEnabled: false
+rootNamespace: "kube-system"
+
+replicaCount: 2
+
+runtimeStatusIntegrator:
+  image:
+    registry: quay.io
+    repository: sysdig/runtime-status-integrator
+    pullPolicy: IfNotPresent
+    tag: "0.1.2-v42d4ca4"
+
+  # Params to manage leader election
+  # Leader election is implemented leveraging the native capabilities of Kubernetes
+  # see: https://kubernetes.io/blog/2016/01/simple-leader-election-with-kubernetes/
+  # leaderElectionLeaseNameOverride: ""
+  # leaderElectionLeaseNamespaceOverride: ""
+  multiCluster:
+    # kubeconfigSecretName: ""
+
+  localCluster:
+    # restrict access to specific docker secrets to obtain pull credentials. Optional.
+    # Default behaviour is listing all secrets
+    # rbac:
+    #   allowedPullSecrets:
+    #     namespaceOneName:
+    #       - myDockerSecretOne
+    #       - myDockerSecretTwo
+    #     anotherNamespace:
+    #       - myOtherDockerSecretOne
+    #       - myOtherDockerSecretTwo
+
+  ports:
+    metrics: 25000
+    probes: 7000
+  resources:
+    limits:
+      cpu: "1"
+      memory: 350Mi
+    requests:
+      cpu: "350m"
+      memory: 350Mi
+
+  natsJS:
+    user: "default-user"
+    # Default uses a random password for each installation.
+    # password: ""
+
+imageSbomExtractor:
+  image:
+    registry: quay.io
+    repository: sysdig/image-sbom-extractor
+    pullPolicy: IfNotPresent
+    tag: "0.1.2-vb7fceb1"
+  ports:
+    metrics: 25001
+    probes: 7001
+  resources:
+    limits:
+      cpu: "1"
+      memory: 350Mi
+    requests:
+      cpu: "150m"
+      memory: 350Mi
```

Comment:
Is it correct to set request & limit in the chart as default? How do we get this values?